### PR TITLE
[full-ci][tests-only] Remove @skipOnRevaMaster tag from ocis

### PR DESCRIPTION
**Pull request by amrita-shrestha**
## Description
This PR remove @skipOnRevaMaster tag from tests code because ocis tests doesn't run on reva master branch

## Related Issue

- https://github.com/owncloud/ocis/issues/5337

## How Has This Been Tested?

- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/package.json
Comment (phil-davis) [inline tests/package.json:1]: Please write something in the PR about why this file is added.

Comment (amrita-shrestha) [reply tests/package.json:1]: my PR should not add this by mistake it is added

**Review by phil-davis — APPROVED** [review]

File: tests/package.json
Comment (phil-davis) [reply tests/package.json:1]: The file is still here.

**Review by phil-davis — CHANGES_REQUESTED** [review]

File: tests/package.json
Comment (phil-davis) [reply tests/package.json:1]: The file is still here @amrita-shrestha

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/36804/50/5
```
       Failed step: When user "Alice" uploads file "filesForUpload/zerobyte.txt" to "textfile.txt" using the TUS protocol on the WebDAV API

        Server error: `POST https://ocis-server:9200/remote.php/dav/spaces/de166e33-ba92-4fad-8716-efaa6f657100$0e2f9596-456e-4cda-92d7-91f0231a8fc8/` resulted in a `500 Internal Server Error` response (GuzzleHttp\Exception\ServerException)
```
I restarted CI.
Is that a flaky test?

**Review by phil-davis — APPROVED** [review]

**Pull request merged**